charts/daemonset/templates/cluster_role.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.clusterrole.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "app.fullname" . }}
+rules:
+{{- toYaml $.Values.clusterrole.rules  | nindent 2 }}
+{{- range $v := .Values.clusterrole.rules }}
```

Comment:
Maybe I am reading this wrong but in a range don't you normally need to use the `$v` ?  Not understanding how the values from clusterrole.rules makes it into the final yaml that gets applied.